argo-apps/tooling-app-of-apps/values.yaml

```diff
@@ -14,17 +14,30 @@ applications:
   #    path: camelk
    - name: nexus
      path: nexus
+     destination: tooling
    - name: amq7
      path: amq7
+     destination: tooling
    - name: sealed-secrets
      path: sealed-secrets/overlays/default
+     destination: tooling
    - name: namespaces
      path: namespaces  
+     destination: tooling
    - name: gitea
      path: gitea
+     destination: tooling
    - name: camelk-community
-     path: camelk-community  
+     path: camelk-community
+     destination: camelk-instructor
    - name: tekton
      path: tekton  
-#    - name: devspaces
-#      path: devspaces  
+     destination: tooling
+   - name: devspaces-install
+     path: devspaces-install
+     destination: tooling
+   - name: devspace-seed
+     path: devspace-seed
```

Comment:
Probably should be devspace**s**-seed , which might also result in renaming app directory itself.
And can we also add "destination: tooling" ?